### DISCUSSION TITLE
Update with zwave-js v12.5.6

### DISCUSSION THREAD

**Discussion by kilian-f-za** [zwave-js/zwave-js-ui#3686] [General]
Hello, when will a new release come with the integration of ZWAVE-JS V12.5.6? This release includes an important config file update for me. Thanks.

**Comment by robertsLando** [comment]
@kilian-f-za I plan to do one pn Monday, in the meanwhile if you are using docker you can switch to master tag instead of latest or just rub a config update from the ui to update only the config db
